test/core/util/run_with_poller.sh

```diff
@@ -16,4 +16,17 @@
 set -ex
 export GRPC_POLL_STRATEGY=$1
 shift
+blackhole_address_setting="$1"
```

Comment:
I am not a fan of adding custom logic to the helper .sh scripts. Over time this is going to snowball and we're gonna end up with maintenance hell and a setup that so complex that no one understand it (and yes, it happened to use before - just look at all the .py and .sh testing scripts we have).